src/app/lib/utilities/getUrlHelpers/getMostReadUrls/index.js

```diff
@@ -1,7 +1,14 @@
-export const getMostReadEndpoint = ({ service, variant }) =>
-  variant
+export const getMostReadEndpoint = ({ service, variant, isBff = false }) => {
+  if (isBff) {
+    return variant
+      ? `/fd/simorgh-bff?pageType=mostRead&service=${service}&variant=${variant}`
+      : `/fd/simorgh-bff?pageType=mostRead&service=${service}`;
```

Comment:
Is it mainly for E2Es that we're using the public CDN version of the BFF url here?

Reply:
It's also for the actual MostRead page - which is the current behaviour - using the web cdn with the mostread.json endpoint - https://github.com/bbc/simorgh/blob/561414ae7c1e6636372381a8e0deddf48f926c1c/envConfig/live.env#L19